asrtoolkit/convert_transcript.py

```diff
@@ -3,43 +3,34 @@
 Python class for converting file formats used in Automatic Speech Recognition
 """
 
-import argparse
 import logging
 import sys
 
+from fire import Fire
+
 from asrtoolkit.file_utils.script_input_validation import assign_if_valid
 
 LOGGER = logging.getLogger(__name__)
 
 
 def check_input_file_validity(input_file):
     if input_file is None:
-        LOGGER.error("Invalid input file {}".format(input_file))
+        LOGGER.error("Invalid input file %s", input_file)
         sys.exit(1)
 
 
 def convert(input_file, output_file):
+    """
+    Convert between text file formats
```

Comment:
Again, we should probably have a blurb about argument information here.

Reply:
After learning, Fire gives enough information about input type.

Should we mention the valid formats here or are they elsewhere in the docs?

Reply:
fixed in 128cf10